source/_integrations/tod.markdown

```diff
@@ -24,6 +24,8 @@ On time:
   description: The time when the sensor should turn on.
 Off time:
   description: The time when the sensor should turn off.
+Unique id:
+  description: Provide a unique id to support customization
```

Comment:
The UI configuration (config flow) doesn't provide this field (and doesn't need it).

```suggestion
```